### PR TITLE
Coinbase (purchase flow): adds bitcoin fee to create transaction

### PR DESCRIPTION
**Pull request by cmgustavo**
* Check if amount is enough to cover the bitcoin fee
* Reduce amount to pay bitcoin fee

Note: After buy bitcoin through Copay integration, in Coinbase side, it tries to create a transaction to transfer the bitcoins bought to your Copay wallet. As all bitcoin transactions, these have a network fee that must be paid with your funds.

### CODE REVIEW

File: src/js/services/coinbaseService.js [hunk 1]
```diff
@@ -107,6 +107,30 @@ angular.module('copayApp.services').factory('coinbaseService', function($http, $
...
+    feeService.getFeeRate(null, 'normal', function(err, feePerKB) {
+      if (err) return cb(err);
+      var feeBTC = (feePerKB * txNormalFeeKB / 100000000).toFixed(8);
+
+      return cb(null, amount - feeBTC, feeBTC);
+    });
+  };
+
+  root.checkEnoughFundsForFee = function(amount, cb) {
+    root.getReductedAmountByFee(amount, function(err, reductedAmount) {
```
Comment (matiu) [inline src/js/services/coinbaseService.js]: get Reduced Amount By Fee
or 
getAmountLessFee

Comment (matiu) [inline src/js/services/coinbaseService.js]: or getNetAmount

Comment (cmgustavo) [inline src/js/services/coinbaseService.js]: Updated. Thanks.

**Comment by matiu** [comment]
code ACK

**Pull request merged**